packages/story-editor/src/app/media/utils/useMediaUploadQueue/test/useMediaUploadQueue.js

```diff
@@ -97,6 +109,15 @@ describe('useMediaUploadQueue', () => {
         isTranscoding: false,
         isMuting: false,
         isTrimming: false,
+        isResourceMuting,
+        isResourceMutingById,
+        isResourceProcessing,
+        isResourceProcessingById,
+        isResourceTranscoding,
+        isResourceTranscodingById,
+        isResourceTrimming,
+        isResourceTrimmingById,
+        isResourceUploadingById,
```

Comment:
It's a bit pointless to get functions from `result.current.state` and then compare them with `result.current.state`...

I'd rather use `expect.any(Function)` here.

```suggestion
        isResourceMuting: expect.any(Function),
        isResourceMutingById: expect.any(Function),
        isResourceProcessing: expect.any(Function),
        isResourceProcessingById: expect.any(Function),
        isResourceTranscoding: expect.any(Function),
        isResourceTranscodingById: expect.any(Function),
        isResourceTrimming: expect.any(Function),
        isResourceTrimmingById: expect.any(Function),
        isResourceUploadingById: expect.any(Function),
```

Same for the test below